ui/app/src/components/MediaCard/MediaCard.tsx

```diff
@@ -109,7 +109,11 @@ function MediaCard(props: MediaCardProps) {
     viewMode = 'card'
   } = props;
   // endregion
-  const { name, path, type } = item;
+  let { name, path, type } = item;
+  // TODO: search item type comes as 'Other' in audio files, it needs to be updated to 'Audio'
```

Comment:
This is really more of a backend Todo, innit? If so, let's remove this ToDo and make sure to start the discussion and create necessary tickets for the back.

Reply:
Yeah only backend. Removing